src/response.rs

```diff
@@ -102,7 +102,11 @@ pub enum Response {
     // ===== Account =====
     //
     /// Returns a success or failure status of putting a new account.
-    PutAccount(Result<()>),
+    CreateAccount(Result<()>),
+    /// Returns a success or failure status of creating an account for someone else.
+    CreateAccountFor(Result<()>),
+    /// Returns a success or failure status of updating an existing account.
+    UpdateAccount(Result<()>),
     /// Returns an encrypted account packet
     GetAccount(Result<Vec<u8>>),
```

Comment:
This could be a pair of `(Vec<u8>, Signature)`.  The client will then check the signature when the response arrives.